CONTRIBUTING.md

```diff
@@ -103,11 +103,13 @@ To test your changes locally, you can run the project from IntelliJ or gradle.
   - Requires `sam` CLI to be on your `$PATH`.
 - To run **GUI tests**:
   ```
-  ./gradlew guiTest
+  ./gradlew uiTestCore
   ```
   - To debug GUI tests,
     1. Set `runIde.debugOptions.enabled=true` in the gradle file.
```

Comment:
Is this right? It also probably easier to just say run `./gradlew runIdeForUiTests --debug-jvm`

Reply:
This is correct, and yep that is easier, changed